db/migrate/20250504183911_remove_short_name_from_organizations.rb

```diff
@@ -0,0 +1,13 @@
+class RemoveShortNameFromOrganizations < ActiveRecord::Migration[8.0]
+  def up
+    safety_assured do
+      remove_index :organizations, :short_name
```

Comment:
FYI you shouldn't need a separate call to remove the index - once the column is gone, the index should also just disappear.